superduper/backends/base/query.py

```diff
@@ -164,15 +220,16 @@ class Query(_BaseQuery):
     """
 
     flavours: t.ClassVar[t.Dict[str, str]] = {}
-    methods_mapping: t.ClassVar[t.Dict[str, str]] = {}
 
     table: str
     parts: t.Sequence[t.Union[t.Tuple, str]] = ()
     identifier: str = ''
 
     def __getitem__(self, item):
+        if isinstance(item, str):
+            return QueryCondition(item)
         if not isinstance(item, slice):
-            return super().__getitem__(item)
+            raise TypeError('Query index must be a string or a slice')
```

Comment:
The parent class does not define the __getitem__ method.

Reply:
yes